core/iwasm/compilation/aot_compiler.c

```diff
@@ -4323,7 +4326,11 @@ aot_emit_object_file(AOTCompContext *comp_ctx, char *file_name)
                      file_name, asm_file_name);
             LOG_VERBOSE("invoking external ASM compiler:\n\t%s", cmd);
 
+#if !(defined(_WIN32) || defined(_WIN32_))
             ret = system(cmd);
+#else
+            ret = _spawnlp(_P_WAIT, "cmd.exe", "/c", cmd, NULL);
+#endif
```

Comment:
Maybe we can wrap an API since there are several similar places doing like this. For example, in core/iwasm//shared/utils/bh_common.{c|h}, add bh_system?